customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDb.java

```diff
@@ -1,14 +1,15 @@
 package no.unit.nva.customer.model;
 
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import no.unit.nva.customer.model.interfaces.Customer;
 import nva.commons.utils.JacocoGenerated;
 
 import java.time.Instant;
 import java.util.Objects;
 import java.util.UUID;
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public class Customer {
+@JsonTypeName("Customer")
+public class CustomerDb implements Customer {
```

Comment:
open question: isn't what is stored here really a model part of the ontology?  (IRI of Customer). 

Thinking of all the `With*` interfaces defined in the nva-datamodel-java. I guess we have those for having to only declaring a serialization mode once and we scope access with the `With*` interfaces?

Reply:
But since `Customer` ownership belongs to this microservice, I kinda feels right with having it simply live here. 
So as long as no other services requires to work with `Customer` .

Reply:
We don't have an ontology for Customer yet, let's create one in another task.